NetKAN/TweakScaleCompanion.netkan

```diff
@@ -0,0 +1,31 @@
+spec_version: v1.8
+identifier: TweakScaleCompanion
+name: TweakScale Companion
+abstract: >-
+  **TweakScale Companion Program** is a series of "Add'On's Add'Ons",
+  adding TweakScale /L support for third parties but decoupling the
+  target Add'On's life cycle from TweakScale's, easing maintenance
+  and distribution efforts. This package aims to make TweakScale
+  user's life easier at first installs with a (most of the time)
+  updated collection with all the currently available Companions
+  at once.
```

Comment:
The `abstract` should be a **short** description, a brief sentence fragment to help the user understand what the mod is. This is much longer than it really should be.

Reply:
Solved on commit https://github.com/KSP-CKAN/NetKAN/pull/9536/commits/21b0d112e86019e628d56c9abb8aa98b15316372

Reply:
Amended on commit https://github.com/KSP-CKAN/NetKAN/pull/9536/commits/3b8a6c0e0bd55088dbbff7be6a7591f117382feb